doc/classes/Sprite3D.xml

```diff
@@ -96,21 +98,28 @@
 	</methods>
 	<members>
 		<member name="frame" type="int" setter="set_frame" getter="get_frame">
+			Current frame to display from sprite sheet. [code]vframes[/code] or [code]hframes[/code] must be greater than 1.
 		</member>
 		<member name="hframes" type="int" setter="set_hframes" getter="get_hframes">
+			The number of horizontal frames in the sprite sheet.
 		</member>
 		<member name="region_enabled" type="bool" setter="set_region" getter="is_region">
+			If [code]true[/code] texture will be cut from a larger atlas texture. See [code]region_rect[/code]. Default value: [code]false[/code].
```

Comment:
Avoid future too. It adds extra words.
`If [code]true[/code], the node's texture is cut from a texture atlas. See [member region_rect]. Default value: [code]false[/code].